### PR TITLE
Lemnisk latest doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/lemnisk/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Lemnisk"
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/index.mdx:2]: Can we convert it into Lemnisk Marketing Automation?

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+### Property mappings
+
+The following table lists the mappings between the RudderStack and Lemnisk properties:
+
+| RudderStack property   | Lemnisk property  | Data type |
+| :-------------| :-----------------| :-----------------| 
+|  `userId` <br/> `anonymousId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `userId` | String |
+|  `email` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `email` | String |
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:36]: We can remove email mapping as we released a fix for this afterwards

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+| `properties` | `properties` | Object |
+| `event`  | `eventname` | String |
+| `track` | `type` | String |
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:39]: We can remove type property as well.

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+### Property mappings
+
+The following table lists the mappings between the RudderStack and Lemnisk properties:
+
+| RudderStack property   | Lemnisk property  |  Data type |
+| :-------------| :-----------------| :-----------------| 
+|  `userId` <br/> `anonymousId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `userId` <br/> `id` | String |
+| `type` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `type` | String |
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:69]: Same as above

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+### Property mappings
+
+The following table lists the mappings between the RudderStack and Lemnisk properties:
+
+| RudderStack property   | Lemnisk property  | Data type |
+| :-------------| :-----------------| :-----------------| 
+|  `userId` <br/> `anonymousId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `userId` <br/> `id` | String |
+| `name` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `name` | String |
+| `type` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `type` | String |
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:121]: Same as above

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Web Cloud Mode
+- **Write Key**: Enter the unique write key of your Lemnisk data source.
+- **Region URL**: Enter the Lemnisk region URL.
+
+#### Server-side Cloud Mode
+- **X-API-Key**: Enter the Lemnisk X-API-Key.
+- **X-API-PassKey**: Enter the Lemnisk X-API-PassKey.
+- **Source ID**: Enter the Lemnisk Source ID.
+
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:33]: We have some more fields as well which are Region URL and Write Key

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+<div class="infoBlock">
+You may contact your Lemnisk customer success manager to obtain any of the above details.
+</div>
+
+To send events to Lemnisk via the <Link to="/destinations/rudderstack-connection-modes/#device-mode">device mode</Link>, you need to configure the following settings:
+
+#### Web Device Mode Settings
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:40]: We have one field here that is the Client Id which may be referred as AccountId by Lemnisk sometimes

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+    <div class="infoBlock">
+    You may contact your Lemnisk customer success manager to obtain any of the above details.
+    </div>
+
+    #### None of the above (Select this if you want to use Device Mode only)
+    - **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Lemnisk. For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
+    - **Client ID**: Enter the unique ID generated for your Lemnisk account. You may contact your Lemnisk customer success manager to obtain the client ID.
+    - **Use device mode to send events:** Make sure this option is enabled if you want to send events using the <Link to="/destinations/rudderstack-connection-modes/#device-mode">device mode</Link>.
+    - **Write Key**: Enter the unique write key of your Lemnisk data source.
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:44]: This `Write Key` and the above `Client Id` and `Use Device Mode... ` are visible only when connected to JS as a source that is web device ode Settings

**Review by anantjain45823 — APPROVED** [review]
Just check up on the Last comment if it is suitable or not

**Review by anantjain45823 — APPROVED** [review]

File: docs/destinations/streaming-destinations/lemnisk/index.mdx [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+description: Send your event data from RudderStack to Lemnisk.
+aliases: ["Lemnisk Marketing Automation", "Lemnisk CDP"]
+---
+
+[Lemnisk](https://www.lemnisk.co/) is a Customer Data Platform which provides personalization and real-time marketing automation solution. It aims to deliver superior customer experiences that result in increased conversions, retention, and growth for enterprises.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/index.mdx:7]:
```suggestion
[Lemnisk](https://www.lemnisk.co/) is a Customer Data Platform that provides personalization and real-time marketing automation solutions. It delivers superior customer experiences that increase user conversions, retention, and growth.
```

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+description: Send events to Lemnisk using RudderStack cloud mode.
+---
+
+RudderStack lets you send your event data to Lemnisk via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>. Find the open source code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/src/v0/destinations/lemnisk">GitHub repository</a>.
+
+RudderStack supports the following Lemnisk platforms:
+
+## DIAPI
+
+When you choose to send events to Lemnisk via the server-side cloud mode, Lemnisk' [HTTP API](https://lemnisk.gitbook.io/lemnisk-platform-documentation/sources/http-api) is used to send your website or app data.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:12]:
```suggestion
When you choose to send events to Lemnisk via the server-side cloud mode, RudderStack uses Lemnisk's [HTTP API](https://lemnisk.gitbook.io/lemnisk-platform-documentation/sources/http-api) to send your website or app data.
```

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+The following table lists the mappings between the RudderStack and Lemnisk properties:
+
+| RudderStack property   | Lemnisk property  | Data type |
+| :-------------| :-----------------| :-----------------| 
+|  `userId` <br/> `anonymousId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `userId` | String |
+| `properties` | `properties` | Object |
+| `event`  | `eventname` | String |
+| `config.srcid` | `srcId` | String |
+| `config.diapiWriteKey`  | `writekey` | String |
+
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:40]:
```suggestion
```

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+
+## Pixel Listener
+
+When you choose to send events to Lemnisk via the web cloud mode, Lemnisk' [JavaScript SDK](https://lemnisk.gitbook.io/lemnisk-platform-documentation/sources/javascript-sdk) is used to send data from your website.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:44]:
```suggestion
When you choose to send events to Lemnisk via the web cloud mode, RudderStack uses Lemnisk's [JavaScript SDK](https://lemnisk.gitbook.io/lemnisk-platform-documentation/sources/javascript-sdk) to send data from your website.
```

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+
+## Identify
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:46]:
```suggestion
### Identify
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:46]: Is this call supported only for Pixel Listener?

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+The following table lists the mappings between the RudderStack and Lemnisk properties:
+
+| RudderStack property   | Lemnisk property  |  Data type |
+| :-------------| :-----------------| :-----------------| 
+|  `userId` <br/> `anonymousId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `userId` <br/> `id` | String |
+| `context` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `context` | Object |
+| `traits` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `customerProperties` | Object |
+| `messageId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `messageId` | String |
+
+## Track
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:70]:
```suggestion
### Track
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:70]: Same as `identify`.

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+A sample `identify` call is shown below:
+
+```javascript
+rudderanalytics.identify("hashed_user_id", {
+  name: "Alex Keener",
+  email: "alex@example.com",
+})
+```
+
+### Property mappings
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:59]: Should be H4 if that's the case.

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 16]
```diff
@@ -0,0 +1,120 @@
...
+| :-------------| :-----------------| :-----------------| 
+|  `userId` <br/> `anonymousId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `userId` <br/> `id` | String |
+| `event` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `event` | String |
+| `properties`   | `properties` | Object |
+| `timestamp` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `originalTimestamp` | String |
+| `messageId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `messageId` | String |
+| `context` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `context` | Object |
+| `properties`   | `properties` | Object |
+
+## Page
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:99]: same as above.

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-device-mode.mdx [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+description: Send events to Lemnisk using RudderStack web device mode.
+---
+
+RudderStack lets you send your event data to Lemnisk via the <Link to="/destinations/rudderstack-connection-modes/#device-mode">device mode</Link> using the native web SDK.
+
+## Identify
+
+You can use the <Link to="/event-spec/standard-events/identify">`identify`</Link> call to identify a user and associate them to their actions. It also lets you record any traits about them like their name, email, etc.
+
+When you send an `identify` event via Rudderstack, Lemnisk's [`identify`](https://lemnisk.gitbook.io/lemnisk-platform-documentation/events/identify) event is triggered. Lemnisk needs a unique user ID to associate an event with a user. By default, Lemnisk uses a cookie to identify the user. Optionally, a unique user ID can be passed in the first parameter.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-device-mode.mdx:12]:
```suggestion
When you send an `identify` event via Rudderstack, Lemnisk's [`identify`](https://lemnisk.gitbook.io/lemnisk-platform-documentation/events/identify) event is triggered. Lemnisk needs a unique user ID to associate an event with a user. By default, it uses a cookie to identify the user. Optionally, you can also pass a unique user ID in the first parameter.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/lemnisk/lemnisk-device-mode.mdx:12]: "in the first parameter.." could be confusing. LMK if the above change makes sense.

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+
+    #### Server-side Cloud Mode
+    - **X-API-Key**: Enter the Lemnisk X-API-Key.
+    - **X-API-PassKey**: Enter the Lemnisk X-API-PassKey.
+    - **Write Key**: Enter the unique write key of your Lemnisk data source.
+    - **Source ID**: Enter the Lemnisk Source ID.
+    - **Region URL**: Enter the Lemnisk region URL.
+
+    <div class="infoBlock">
+    You may contact your Lemnisk customer success manager to obtain any of the above details.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:37]: can we add some contact like email, or their contact page here?

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+    </div>
+
+    #### None of the above (Select this if you want to use Device Mode only)
+    - **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Lemnisk. For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
+    - **OneTrust Cookie Categories**: This setting lets you associate the <Link to="/sources/sdks/rudderstack-javascript-sdk/consent-managers/onetrust/">OneTrust</Link> cookie consent groups to Lemnisk.
+
+    The below settings are visible only when a JavaScript source is connected to Lemnisk:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:44]:
```suggestion
The below settings are visible only when a JavaScript source is connected to Lemnisk: 
```

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+    <div class="infoBlock">
+    You may contact your Lemnisk customer success manager to obtain any of the above details.
+    </div>
+
+    #### None of the above (Select this if you want to use Device Mode only)
+    - **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Lemnisk. For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
+    - **OneTrust Cookie Categories**: This setting lets you associate the <Link to="/sources/sdks/rudderstack-javascript-sdk/consent-managers/onetrust/">OneTrust</Link> cookie consent groups to Lemnisk.
+
+    The below settings are visible only when a JavaScript source is connected to Lemnisk: 
+    - **Client ID**: Enter the unique ID generated for your Lemnisk account. You may contact your Lemnisk customer success manager to obtain the client ID.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:45]:
```suggestion
    - **Client ID**: Enter the unique ID generated for your Lemnisk account. Contact your Lemnisk customer success manager to obtain the client ID.
```

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx (hunk 18 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:37]:
```suggestion
    Contact your Lemnisk customer success manager to obtain any of the above details.
```

File: docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx [hunk 21]
```diff
@@ -0,0 +1,120 @@
...
+```javascript
+rudderanalytics.track("Product Reviewed", {
+  review_id: "12345",
+  product_id: "123",
+  rating: 3.0,
+  review_body: "Average product, expected much more."
+})
+```
+
+### Property mappings
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/lemnisk/lemnisk-cloud-mode.mdx:29]: Should be H4 as it's associated with `track`, IMO

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx (hunk 19 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:44]: This is the part of cloud mode setting, hence indented.

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx (hunk 18 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx:37]: I am not sure if their contact page is relevant for this one as they might have a customer success manager already assigned. Also, in the external contribution PR, there was no link/ref provided for CSM. LMK what do you think

**Review by ameypv-rudder — APPROVED** [review]

File: docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx (hunk 18 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/lemnisk/setting-up-lemnisk.mdx]: yep, makes sense. Approved the PR!

**Pull request merged**